.editorconfigSortertGammel

```diff
@@ -0,0 +1,279 @@
+
+
+
+
+
+
+[*.java]
+[*.json]
+[*.md]
+[*.{yml,yaml}]
+[*]
+[pom.xml]
+charset = utf-8
+charset = utf-8
+end_of_line = lf
```

Comment:
Ulempen med denne approachen er at man mister kontekst over hvilken fil typer dette gjelder.

Reply:
Vi kunne kanskje starte med `[*.java]` evt `[pom.xml]` først?

Reply:
Det er ikke meningen at  filen skal være sortert alfabetisk når vi bruker den. (Tror ikke konfigen vil funke slik den er nå uansett). Jeg har bare sortert den her fordi Jens Otto ville se diffen mellom gammel og ny config, så denne branchen skal aldri merges. Grunnen til at jeg har lagd PR, er fordi github sin markdown har støtte for patch-notasjon som gjør at det er enkelt å lese diffen. 

Så konfigen vi kommer til å ende opp med vil være seksjonert på filtype slik den orginalt er i dag 😄